R/util_assert.R

```diff
@@ -5,3 +5,12 @@ assert_has_columns <- function(x, columns, name = deparse(substitute(x))) {
                  name, paste(msg, collapse = ", ")))
   }
 }
+
+assert_has_files <- function(x, files) {
+  msg <- setdiff(files,x)
+  if (length(msg) > 0L) {
+    stop(sprintf("Missing file '%s' from: %s.",
```

Comment:
this will behave badly if there are more than one missing file (try it!).  Once you've tried it can you write some unit tests for practice?